pkgs/default.nix

```diff
@@ -148,8 +148,9 @@ in rec {
 
   get-user-data = pkgs.writeScriptBin "get-user-data" ''
     #! /bin/sh
-    exec ${retry}/bin/retry \
-      ${pkgs.curl}/bin/curl -sf http://169.254.169.254/latest/user-data
+    exec ${pkgs.curl}/bin/curl \
```

Comment:
Alright, I've updated it to use `wget`. Can you elaborate or point to some resources about why curl's retry sucks?